### PR TITLE
Update README.md: api key config

### PR DESCRIPTION
**Pull request by ankitkat042**
added how to add key configurations both in Windows and Linux to avoid confusion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ GPT Engineer is made to be easy to adapt, extend, and make your agent learn how
 **Setup**:
 
 - `pip install -r requirements.txt`
-- `export OPENAI_API_KEY=[your api key]` with a key that has GPT4 access
+- Configure your OpenAI API key:
+    - On Linux: `export OPENAI_API_KEY=[your api key]` in terminal.  
+    - On Windows: edit `main.py` file and add this line - `open.api_key = "your api key"`.
```
Comment (profplum700) [inline README.md:24]: Can you provide an example of where exactly to place it too?

Comment (profplum700) [inline README.md:24]:
```suggestion
    - On Windows: edit `main.py` file and add this line - `openai.api_key = "your api key"`.  
```

**Comment by patillacode** [comment]
This will probably be outdated if for example #29 is merged

**Comment by AntonOsika** [comment]
Let's just add explanation for how to add env vars in window!

File: README.md (hunk 1 above)
Comment (Dkv9) [reply README.md:24]: `NameError: name 'openai' is not defined. Did you mean: 'open'?`

This error occurs.
set, export, and putting `openai.api_key = "your api key"` into main.py
all of them are currently not working on Windows.

**Pull request closed without merging**